src/libraries/System.Private.CoreLib/src/System/Decimal.cs

```diff
@@ -753,7 +752,7 @@ public static int ToInt32(decimal d)
             if ((d.High | d.Mid) == 0)
             {
                 int i = (int)d.Low;
-                if (!d.IsNegative)
+                if (!IsNegative(d))
```

Comment:
If you're not removing the `IsNegative` property, why change these callsites?

Reply:
Its changed from an `internal bool IsNegative { get; }` instance property to a `public static bool IsNegative(decimal d)` static method.

Reply:
Ah, for some reason `Decimal.DecCalc.cs` sorted above `Decimal.cs`, so when I went looking for the property removal explicitly I didn't find it.  Now I see it, and withdraw the question.